api/src/services/supertokens.js

```diff
@@ -0,0 +1,20 @@
+import * as emailpassword from "supertokens-node/recipe/emailpassword";
+import * as sessions from "supertokens-node/recipe/session";
+
+export const config = {
+  appInfo: {
+    apiDomain: "http://localhost:8910/",
+    appName: "SuperTokens RedwoodJS",
+    websiteDomain: "http://localhost:8910/",
+    // apiBasePath: "/.netlify/functions/"
+  },
+  supertokens: {
+    connectionURI: "try.supertokens.io"
+  },
+  recipeList: [
+    emailpassword.init(),
+    sessions.init()
+  ],
+  apiWebProxyPath: "/.netlify/functions",
+  isInServerlessEnv: true
+}
```

Comment:
This is the supertokens config. It is used by the `api/src/functions/auth.js` serverless function and also used by the `api/src/functions/graphql.js` serverless function.